compiler/one-cmds/onelib/export_constant.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from constant import CONSTANT
+
+import argparse
+import configparser
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Export CONSTANT value with given file format')
+    parser.add_argument(
+        '-c', '--constant', type=str, required=True, help='Constant name to export')
+    parser.add_argument(
+        '-f',
+        '--format',
+        type=str,
+        required=True,
+        choices=['cfg', 'txt'],
```

Comment:
How about `optcfg` or `opt_cfg`?

```suggestion
        choices=['opt_cfg', 'txt'],
```

The current implementation uses a fixed section name [one-optimize].

Reply:
Hmm.. I think `opt_cfg` is also hard to know exporting cfg has only [one-optimize] section. I'll append the related help message to the description.